src/haversine.jl

```diff
@@ -33,11 +33,12 @@ function (dist::Haversine)(x, y)
     2 * dist.radius * asin( min(√a, one(a)) ) # take care of floating point errors
 end
 
-haversine(x, y, radius::Real = Float32(6371000)) = Haversine(radius)(x, y)
+haversine(x, y, radius=Float32(6371000)) = Haversine(radius)(x, y)
 
 @noinline haversine_error(dist) = throw(ArgumentError("expected both inputs to have length 2 in $dist distance"))
 
-
+_deg2rad(x::Real) = deg2rad(x)
+_deg2rad(x) = x # targets unitful numbers
```

Comment:
Maybe at least continue requiring `::Number`?

Also, is this the only place where we required `::Real`?

Reply:
The only other hits for `::Real` I found are `p` in `*Minkowski` and `alpha` in `RenyiDivergence`, which both seem reasonable to me.